app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -136,11 +138,17 @@ firewall --disable
 <% else -%>
 firewall --<%= os_major >= 6 ? 'service=' : '' %>ssh
 <% end -%>
+
+<% if host_param_true?('disable-kdump') -%>
+%addon com_redhat_kdump --disable
+<% end -%>
+
```

Comment:
I'm not sure the diff shows well, but I prefer to have the rendered output to be readable. That means I prefer the empty line to be inside the conditional
```suggestion

<% end -%>
```

Or more explicit:
```erb
<% if host_param_true?('disable-kdump') -%>
%addon com_redhat_kdump --disable

<% end -%>
<% if (is_fedora && os_major >= 28) || (rhel_compatible && os_major > 7) -%>
```

Reply:
Done, even though it'd be nice to have something to divide different IFs, maybe an empty comment `<%# >` might be the solution?

Reply:
Recheck now, maybe this could be a solution? Adding comment should allow to divide blocks while keeping readability in both template and output